awesomeConfig.cmake

```diff
@@ -77,7 +77,7 @@ if(GENERATE_MANPAGES)
 endif()
 
 if(GENERATE_DOC)
-    if(NOT LDOC_EXECUTABLE)
+    if(NOT LDOC_EXECUTABLE OR DO_COVERAGE)
```

Comment:
Should not check for DO_COVERAGE here and above: the build should (and does) not install ldoc instead.